### PR TITLE
Addressing USDOT PR comments 6/13/2024

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
Some comments have been left on https://github.com/usdot-jpo-ode/jpo-ode/pull/547 that need to be addressed.

### Changes
- Updated example docker-compose.yml in `dockerhub.md`
- Removed '=' from kafka script arguments
- Updated SDW references to 'SDX' in comments & documentation
- Updated `quickstart-compose.yml`
- Updated documentation on automatically creating topics w/ Kafka

## Testing
The ODE has been verified to compile with these changes & the unit tests pass.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
This all looks good.

File: kafka.md [hunk 1]
```diff
@@ -41,16 +41,7 @@ If you don't specify a broker id in your docker-compose file, it will automatica
...
-creation, a ```KAFKA_CREATE_TOPICS``` environment variable can be
-added in ```docker-compose.yml```.
-
-Here is an example snippet from ```docker-compose.yml```:
-
-        environment:
-          KAFKA_CREATE_TOPICS: "Topic1:1:3,Topic2:1:1"
-
-```Topic 1``` will have 1 partition and 3 replicas, ```Topic 2``` will have 1 partition and 1 replica.
+If you want to have Kafka automatically create topics during creation, modify the `scripts\kafka\kafka_init.sh` script to include the topics you want to create. The script is run by a kafka init container upon startup. The default script creates a single topic `test` with 1 partition and 1 replica.
```
Comment (mwodahl) [inline kafka.md:44]: Doesn't the kafka-init.sh script create all topics by default instead of just the single 'test' topic?

Comment (dmccoystephenson) [reply kafka.md]: Thanks for catching that! This has been corrected.

**Review by mwodahl — APPROVED** [review]
Looks good!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**